R4ML/R/ml.model.pca.R

```diff
@@ -76,6 +76,7 @@ setClass(
 #'
 #' }
 #'
+#' @import methods
```

Comment:
They have to be in both the DESCRIPTION and in the NAMESPACE. By using `@import` roxygen automatically adds them to the NAMESPACE.